content/infrastructure/features.md

```diff
@@ -11,36 +11,36 @@ weight = 10
 [[feature]]
   icon = "boxes"
   icon_pack = "fas"
-  name = "An environment for your community"
-  description = "2i2c Hubs provide data environments that have been designed for research and education, and can be customized for your community."
+  name = "Custom environments"
+  description = "Python? R? Julia? Use whatever custom data science stack you want."
 
 [[feature]]
   icon = "desktop"
   icon_pack = "fas"
-  name = "Provide a variety of open source interfaces"
-  description = "Use a variety of user interfaces, including Jupyter Notebook, Jupyter Lab, RStudio, VSCode. If it can be run in a browser, it can run on a 2i2c Hub."
+  name = "Data Science UI"
+  description = "JupyterLab, Jupyter Notebook, RStuio - pick tools that fits your workflow."
 
 [[feature]]
-  icon = "book"
+  icon = "key"
   icon_pack = "fas"
-  name = "Connect with your own online content"
-  description = "Provide notebooks, text files, and scripts for your hub's community through links that point to online repositories."
+  name = "Frictionless Authentication"
+  description = "Reduce onboarding friction by logging in your favorite authentication provider."
```

Comment:
I think this is not quite a true statement, no? Don't we only provide real support for OAuth / github / orcid?

Reply:
I think that's true for the pilot repos, but not true elsewhere. For example, the Farallon hub runs a *different flavor* of GitHub auth (without auth0, but with per-organization limits). UToronto uses Microsoft AAD. So I think this is overall an ok claim to make.